pom.xml

```diff
@@ -1214,6 +1215,12 @@
           <version>${version.net.ltgt.gwt.maven}</version>
           <extensions>true</extensions>
         </plugin>
+        <plugin>
+          <groupId>org.kie</groupId>
+          <artifactId>kie-gwthelper-maven-plugin</artifactId>
+          <version>${version.org.kie.gwthelper.maven}</version>
```

Comment:
I believe that the plugin version should be aligned with the rest of KIE projects, so simply `${version.org.kie}`.

Reply:
_done_